.github/workflows/cassandra.yml

```diff
@@ -24,5 +24,11 @@ jobs:
     - name: Run tests on cassandra
       run: |
         CDC='disabled' SCYLLA_URI=172.42.0.2:9042 SCYLLA_URI2=172.42.0.3:9042 SCYLLA_URI3=172.42.0.4:9042 cargo test --verbose -- --skip test_views_in_schema_info
-    - name: Stop cluster
+    - name: Stop the cluster
+      if: ${{ always() }}
+      run: docker-compose -f test/cluster/cassandra/docker-compose.yml stop
```

Comment:
Why did you add the `stop` command? Was this to see all the cluster logs, including those produced during Scylla shutdowns?

Reply:
Yes. I can't do it after `docker-compose shutdown` because it deletes the containers along with the logs.